tests/admctrl_configmap_test.go

```diff
@@ -10,17 +10,23 @@ import (
 	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/admissioncontrol"
 	"github.com/stackrox/rox/pkg/booleanpolicy/fieldnames"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/features"
 	"github.com/stackrox/rox/pkg/gziputil"
 	"github.com/stackrox/rox/pkg/namespaces"
 	"github.com/stackrox/rox/pkg/testutils"
 	"github.com/stackrox/rox/pkg/testutils/centralgrpc"
+	"github.com/stackrox/rox/pkg/uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func TestAdmissionControllerConfigMap(t *testing.T) {
-	t.Skip("ROX-14495: This test is not working.")
+	if env.PostgresDatastoreEnabled.BooleanSetting() {
+		t.Setenv(features.NewPolicyCategories.Name(), "true")
```

Comment:
so basically if this is a test for postgres, then this test will only run the policy category path. But is that accurate? Should it run whether or not policy categories are enabled?

Reply:
In postgres, new policy categories only, so yes. Am I missing anything?

Reply:
( we plan to remove the new categories feature flag soon and keep postgres only - then it really won't matter)